Packs/Palo_Alto_Networks_Enterprise_DLP/Integrations/Palo_Alto_Networks_Enterprise_DLP/Palo_Alto_Networks_Enterprise_DLP.py

```diff
@@ -75,12 +100,42 @@ def _get_dlp_api_call(self, url_suffix: str):
                 break
             count += 1
 
-        if res.status_code < 200 or res.status_code >= 300:
-            raise DemistoException("Request to {} failed with status code {}".format(url_suffix, res.status_code))
-
         result_json = {} if res.status_code == 204 else res.json()
         return result_json, res.status_code
 
+    def _post_dlp_api_call(self, url_suffix: str, payload: Dict = None):
+        """
+        Makes a POST HTTP(s) call to the DLP API
+        Args:
+            url_suffix: URL suffix for dlp api call
+            payload: Optional JSON payload
+        """
+        count = 0
+
+        while count < MAX_ATTEMPTS:
```

Comment:
You actually don't need to use this implementation. `_http_request` has a `retries` arg.
`retries: How many retries should be made in case of a failure. when set to '0'- will fail on the first time`.

Please remove the `while` loop.

Reply:
I tried the approach you suggested and it didn't work. The purpose of the retry is to get a new access_token in case it expired but using _http_request's retries arg made it keep trying with the old access_token. As a result, the POST call always returns 403.